docs/messaging/sns.md

```diff
@@ -0,0 +1,51 @@
+<!--- Hugo front matter used to generate the website version of this page:
+linkTitle: SNS
+--->
+
+# Semantic conventions for AWS SNS
+
+**Status**: [Development][DocumentStatus]
+
+The Semantic Conventions for AWS SNS extend the general
+[AWS SDK Semantic Conventions](/docs/cloud-providers/aws-sdk.md)
+that describe common AWS SDK attributes in addition to the Semantic Conventions
+described on this page.
+
+<!-- semconv span.aws.sns.client -->
+<!-- NOTE: THIS TEXT IS AUTOGENERATED. DO NOT EDIT BY HAND. -->
+<!-- see templates/registry/markdown/snippet.md.j2 -->
+<!-- prettier-ignore-start -->
+<!-- markdownlint-capture -->
+<!-- markdownlint-disable -->
+
+| Attribute  | Type | Description  | Examples  | [Requirement Level](https://opentelemetry.io/docs/specs/semconv/general/attribute-requirement-level/) | Stability |
```

Comment:
are there any general messaging conventions AWS SDK would report on SNS and SQS spans?